src/renderer/components/ft-input/ft-input.vue

```diff
@@ -18,6 +20,12 @@
         :tooltip="tooltip"
       />
     </label>
+    <font-awesome-icon
+      v-if="showClearTextButton"
+      icon="backspace"
+      class="clearInputTextButton"
+      @click="handleClearTextClick"
+    />
```

Comment:
```suggestion
    <font-awesome-icon
      v-if="showClearTextButton"
      icon="backspace"
      class="clearInputTextButton"
      tabindex="0"
      role="button"
      title="$t('Clear Input')"
      @click="handleClearTextClick"
      @keydown.space.prevent="handleClearTextClick"
      @keydown.enter.prevent="handleClearTextClick"
     />
```
This is inaccessible. I suggest you implement these changes here and create a yaml string "Clear Input" as well to use here as the title.